code/game/objects/structures/extinguisher.dm

```diff
@@ -9,7 +9,11 @@
 	integrity_failure = 50
 	var/obj/item/extinguisher/stored_extinguisher

```

Comment:
```DM
var/obj/store_object
```

you'll need to rename all `stored_extinguisher` to `stored_object` tho

Reply:
why though?

Reply:
eh whatever, did it.